reactor-core/src/main/java/reactor/core/publisher/Mono.java

```diff
@@ -632,7 +632,7 @@
 
 	/**
 	 * Aggregate given publishers into a new {@literal Mono} that will be fulfilled
-	 * when all of the given {@literal sources} have been fulfilled. An error will cause
+	 * when all of the given {@literal sources} have received an item. An error will cause
```

Comment:
"produced an item"?

Reply:
good catch was taking the inner perspective somehow

Reply:
still "received an item" instead of "produced an item" 😁